app-devel/xa/autobuild/defines

```diff
@@ -4,3 +4,4 @@ PKGDEP="glibc"
 PKGDES="Cross-assembler for 6502 microprocessor"
 
 MAKE_AFTER="BINDIR=$PKGDIR/usr/bin MANDIR=$PKGDIR/usr/share/man"
+NOPARALLEL=1
```

Comment:
Why not? What's the error? The best practice would be to leave the following comment above this line:

```
FIXME: {A SUMMARY OF THE BUILD ERROR THAT YOU SAW}
```

Reply:
If parallel is not disable, there will be error.The following is the error message.
```
make[1]: Leaving directory '/var/cache/acbs/build/acbs.tfzo6rol/xa-2.4.0/misc'
gcc -o ../xa xa.o xaa.o xal.o xap.o xat.o xar.o xar2.o xao.o xau.o xam.o xacharset.o xalisting.o -lc
make[1]: Leaving directory '/var/cache/acbs/build/acbs.tfzo6rol/xa-2.4.0/src'
[INFO]:  Installing binaries ...
rm -f xa
mkdir -p /var/cache/acbs/build/acbs.tfzo6rol/xa-2.4.0/abdist/usr/bin
mkdir -p /var/cache/acbs/build/acbs.tfzo6rol/xa-2.4.0/abdist/usr/share/man
install xa reloc65 ldo65 file65 printcbm uncpk /var/cache/acbs/build/acbs.tfzo6rol/xa-2.4.0/abdist/usr/bin
install: cannot stat 'xa': No such file or directory
make: *** [Makefile:65: install] Error 1
[ERROR]: Build error detected ^o^
In file included from /usr/bin/autobuild:65
                 from /usr/lib/autobuild3/lib/base.sh:118
                 from /usr/lib/autobuild3/lib/base.sh:122
                 from /usr/lib/autobuild3/proc/50-build_exec.sh:43
/usr/lib/autobuild3/build/29-plainmake.sh:28: In function (unknown):
/usr/lib/autobuild3/build/29-plainmake.sh:28: error: command exited with 2
>  28 | 		|| abdie "Failed to install binaries: $?."
   29 | }

autobuild encountered an error and couldn't continue.
Failed to install binaries: 2.
```

Reply:
The original intention of disabling parallel was to view detailed build logs, but I didn't expect it to succeed.